### ISSUE TITLE
New Rule: Scrollable pane not keyboard accessible

### ISSUE THREAD

**Issue by dsturley** [dequelabs/axe-core#96]
Scrollable areas (overflow: auto / scroll) need a method for keyboard-only users to scroll the area.  This can be fixed if:
1.  The scrollable area has focusable elements within it.
2.  The scrollable area is added to tab order.

**Comment by dylanb** [comment]
Any idea how we get this to perform well?

**Comment by WilcoFiers** [comment]
We have a list of scrollable elements that we need for restoreScroll. We may need a way to bypass the selector/matches and use that list instead. We'll also need to check that there is hidden content and that there is visible content as well. This may have to be experimental.

**Comment by marcysutton** [comment]
Perhaps the new `valid-scrollable-semantics` check could be leveraged for this? https://github.com/dequelabs/axe-core/commit/ce5f3dc765dfb2079424d23a94c64d9043c5a803

**Comment by jkodu** [comment]
# Rule: scrollable-keyboard-accessible

The rule checks that scrollable elements have keyboard access.

**Tags:** wcag2a, experimental

## Matcher

Element(s) that can be scrolled.  (See - `function getScroll`)

## Checks

### Check 1
"any": focusable-content
check that element has focusable content

pass - 
fail - 

### Check 2
"any": is-focusable
check that element is focusable

pass - 
fail -

**Comment by jeankaplansky** [comment]
documented in INR - 6/17/2019

**Comment by jeankaplansky** [comment]
https://dequeuniversity.com/rules/axe/3.3/scrollable-region-focusable

**Comment by jeankaplansky** [comment]
DQU and WCAG links included creating MVP rule help

**Comment by chandana7393** [comment]
Tested, working as expected.
![scrollable_keyboard_Attest](https://user-images.githubusercontent.com/45285854/62292510-2ae76000-b484-11e9-8edf-dcec5af683fb.PNG)
![scrollable_keyboard_Axe](https://user-images.githubusercontent.com/45285854/62292512-2cb12380-b484-11e9-851c-de6f687ffa23.PNG)
![scrollable_keyboard_axe_coconut](https://user-images.githubusercontent.com/45285854/62292517-2e7ae700-b484-11e9-9075-fac7de3798a9.PNG)
**Tested Environment:**
Attest - 2.5.1.21115v
Axe - 3.8.1.21115v
Axe-coconut - 3.8.1.21304v
Chrome - 76.0.3809.87v
Firefox - 68.0.1v
OS - Windows 10 64 bit.
